js_tutorial/exercises/005_prime.js

```diff
@@ -0,0 +1,18 @@
+const readlineSync = require("readline-sync");
+const num = readlineSync.question("Enter the number ");
+const checkIfPrimeNumber = (num) => {
+if (num%num == 0 && num%1 == 0 && num%2 == 1 && num%3 != 0 && num%7 != 0 && num%5 != 0) {
```

Comment:
Logic is not correct.  
A prime number (or a prime) is a natural number greater than 1 that cannot be formed by multiplying two smaller natural numbers. A natural number greater than 1 that is not prime is called a composite number. For example, 5 is prime because the only ways of writing it as a product, 1 × 5 or 5 × 1, involve 5 itself. However, 6 is composite because it is the product of two numbers (2 × 3) that are both smaller than 6. Primes are central in number theory because of the fundamental theorem of arithmetic: every natural number greater than 1 is either a prime itself or can be factorized as a product of primes that is unique up to their order.